packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js

```diff
@@ -105,7 +105,7 @@ class BaseCompare {
 
     // Added to allow for test reusability from terra repositories
     if (specPath.includes('node_modules')) {
-      [, specPath] = specPath.split('node_modules');
+      specPath = specPath.replace('node_modules', path.join('tests', 'wdio', '__snapshots__'));
```

Comment:
Lets remove the __snapshots__ path to avoid adding snapshots to the path twice.

```suggestion
      specPath = specPath.replace('node_modules', path.join('tests', 'wdio'));
```

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/644/commits/b4c3235c97562900a7d405ed3b096bbba318dd8c